{
  "description": "file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.",
  "releases": {
    "bullseye": {
      "fixed_version": "4.2.1-2",
      "repositories": {
        "bullseye": "5.0.10-1"
      },
      "status": "resolved",
      "urgency": "low**"
    },
    "buster": {
      "fixed_version": "4.2.1-2",
      "repositories": {
        "buster": "5.0.4-2"
      },
      "status": "resolved",
      "urgency": "low**"
    },
    "jessie": {
      "fixed_version": "3.4.0-3+deb8u3",
      "repositories": {
        "jessie": "3.4.0-3+deb8u3",
        "jessie-security": "3.4.0-3+deb8u3"
      },
      "status": "resolved",
      "urgency": "low**"
    },
    "sid": {
      "fixed_version": "4.2.1-2",
      "repositories": {
        "sid": "5.0.10-1"
      },
      "status": "resolved",
      "urgency": "low**"
    },
    "stretch": {
      "fixed_version": "4.2.1-2",
      "repositories": {
        "stretch": "4.2.1-2+deb9u1",
        "stretch-security": "4.2.1-2+deb9u1"
      },
      "status": "resolved",
      "urgency": "low**"
    }
  },
  "scope": "remote"
}